lib/matplotlib/dates.py

```diff
@@ -654,9 +654,10 @@ def strftime_pre_1900(self, dt, fmt=None):
         return cbook.unicode_safe(s1)
 
     def strftime(self, dt, fmt=None):
-        """Refer to documentation for datetime.strftime.
+        """
+        Refer to documentation for :meth:`datetime.datetime.strftime`
 
-        *fmt* is a :func:`strftime` format string.
+        *fmt* is a :meth:`datetime.datetime.strftime` format string.
```

Comment:
Probably no need to repeat the same reference twice in two consecutive sentences (e.g. remove the previous line)?  Otherwise looks good.